constellations/clients/orionFakeDataFiller/filler.go

```diff
@@ -20,29 +21,246 @@ import (
+	)
+}
+
+func createProgram(hostAddress, programId string, name string, grade1 string, grade2 string, description string) error {
+	programBody := strings.NewReader(fmt.Sprintf(`{
+		"programId": "%s",
+		"name": "%s",
+		"grade1": %s,
+		"grade2": %s,
+		"description": "%s"
+	}`, programId, name, grade1, grade2, description))
+	fmt.Println(programId + " was created")
+	sendPostRequest(hostAddress+"/api/programs/create", programBody)
+	return nil
+}
+
+func createSemester(hostAddress, semesterId string, title string) error {
+	semesterBody := strings.NewReader(fmt.Sprintf(`{
+		"semesterId": "%s",
+		"title": "%s"
+	}`, semesterId, title))
+	sendPostRequest(hostAddress+"/api/semesters/create", semesterBody)
+	return nil
+}
+
+func createLocation(hostAddress, locationId, street, city, state, zipcode string) error {
+	locationBody := strings.NewReader(fmt.Sprintf(`{
+		"locationId": "%s",
+		"street": "%s",
+		"city": "%s",
+		"state": "%s",
+		"zipcode": "%s"
+	}`, locationId, street, city, state, zipcode))
+	sendPostRequest(hostAddress+"/api/locations/create", locationBody)
+	return nil
+}
+
+func createAchieve(hostAddress, year, message string) error {
+	achieveBody := strings.NewReader(fmt.Sprintf(`{
+		"year": %s,
+		"message": "%s"
+	}`, year, message))
+	sendPostRequest(hostAddress+"/api/achievements/create", achieveBody)
+	return nil
+}
+
+func createAnnounce(hostAddress, author, message, onHomePage string) error {
+	now := time.Now().UTC()
+	nowJson, _ := now.MarshalJSON()
+
+	announceBody := strings.NewReader(fmt.Sprintf(`{
+		"postedAt": %s,
+		"author": "%s",
+		"message": "%s",
+		"onHomePage": %s
+	}`, nowJson, author, message, onHomePage))
+	sendPostRequest(hostAddress+"/api/announcements/create", announceBody)
+	return nil
+}
+
+func createClass(hostAddress, programId, semesterId, classKey, classId, locationId, times string) error {
+	now := time.Now().UTC()
+	nowJson, _ := now.MarshalJSON()
+	var later = now.Add(time.Hour * 24 * 30)
+	laterJson, _ := later.MarshalJSON()
+
+	classBody := strings.NewReader(fmt.Sprintf(`{
+		"programId": "%s",
+		"semesterId": "%s",
+		"classKey": "%s",
+		"classId": "%s",
+		"locationId": "%s",
+		"times": "%s",
+		"startDate": %s,
+		"endDate": %s
+	}`, programId, semesterId, classKey, classId, locationId, times, nowJson, laterJson))
+	sendPostRequest(hostAddress+"/api/classes/create", classBody)
+	return nil
+}
+
+func createSession(hostAddress, classId, cancelled string) error {
+	now := time.Now().UTC()
+	nowJson, _ := now.MarshalJSON()
+	var later = now.Add(time.Hour * 24 * 30)
+	laterJson, _ := later.MarshalJSON()
+
+	sessionBody := strings.NewReader(fmt.Sprintf(`{
+		"classId": "%s"
+		"startsAt": %s
+		"endsAt": %s
+		"cancelled": %s
+	}`, classId, nowJson, laterJson, cancelled))
+	sendPostRequest(hostAddress+"/api/sessions/create", sessionBody)
+	return nil
 }
 
 func sendPostRequest(url string, body io.Reader) {
 	resp, err := http.Post(url, "application/json; charset=UTF-8", body)
```

Comment:
It would be great if we knew which request we're attempting and the "progress" of the script as we're sending so many requests. It would be useful to know which request succeeded and which failed. And if it failed, why.

So like you can have a simple print statement before the post like:
```
Attempting POST to ____________ ...
```
OR
Have a print statement in each of the `create___()` functions.
```
Creating program ap_calc...
Creating program ap_java...
...
```

And so on.